virtual_database_client/handler_pfcwd.go

```diff
@@ -103,12 +103,12 @@ func v2rPortQueuePfcwdStats(path *gnmipb.Path, pathG2S *map[*gnmipb.Path][]table
 		name: "Pfcwd",
 	}
 
-	target_fields := []string{}
-	updatePath(path, &tmpl, parentConfig, leaf, &target_fields)
+	targetFields := []string{}
+	updatePath(path, &tmpl, parentConfig, leaf, &targetFields)
 
 	// Populate tablePaths
-	fmt.Printf("path passed in populate: %v\n", &tmpl)
-	err := pop_PortQueuePfcwdStats(&tmpl, pathG2S, target_fields)
+	//fmt.Printf("path passed in populate: %v\n", &tmpl)
```

Comment:
If you do need keep the debug info, please change to to log message.

Reply:
Adjusted all the fmt.Printf's I could find to logs.

Reply:
Still appears to be a `fmt.Printf)`, and still commented out. Either change to log or remove.

Reply:
Removed.